data/entities.yaml

```diff
@@ -1193,3 +1193,23 @@
     inner workings.
   properties: [portable]
   capabilities: [debug]
+
+- name: victrola
+  display:
+    attr: device
+    char: 'Q'
+  description:
+  - |
+    A device for reading and writing data on circular platters made of
+    a soft plastic material.  The stylus must be made of an extremely
+    hard and durable material.
```

Comment:
Reminds me of [The Portable Phonograph](http://in-sightforum.com/wp-content/uploads/2019/01/The-Portable-Phonograph-L1.pdf) from high school:

> "I have been using thorns as needles," he said. "But tonight, because we have a musician
among us" - he bent his head to the young man, almost invisible in the shadow - "I will use
a steel needle. There are only three left."
>
> The two middle-aged men stared at him in speechless adoration. The one with the big
hands, who wanted to write, moved his lips, but the whisper was not audible.
>
> "Oh, don't," cried the young man, as if he were hurt. "The thorns will do beautifully."
>
> "No," the old man said. "I have become accustomed to the thorns, but they are not really
good. For you, my young friend, we will have good music tonight."
>
> "After all," he added generously, and beginning to wind the phonograph, which creaked,
"they can’t last forever."

Reply:
I wasn't familiar with that story, thanks for the reference!  I just went and read the whole thing.  I wonder if we could somehow stick a reference to the story into the description...